core-blocks/freeform/edit.js

```diff
@@ -114,7 +114,7 @@ export default class FreeformEdit extends Component {
 		} );
 
 		editor.addButton( 'kitchensink', {
-			tooltip: __( 'More' ),
+			tooltip: _x( 'More', 'expand options' ),
```

Comment:
Maybe context should mention "button" for better clarification?

Reply:
Thanks — done